zboss/doc/zboss_certification.rst

```diff
@@ -31,7 +31,8 @@ You can use this platform as the building block for your Zigbee Certified Produc
 Certification IDs
 *****************
 
-Check the compatibility matrices in the `Infocenter`_ for information about the latest certification ID entries (CIDs) for the combination of the ZBOSS stack v3.3 (or newer), the |NCS| version, and Nordic Semiconductor's SoCs:
+Check the compatibility matrices in `TechDocs`_ for information about the latest certification ID entries (CIDs) for the combination of the ZBOSS stack v3.3 (or newer), the |NCS| version, and Nordic Semiconductor's SoCs:
```

Comment:
```suggestion
Check the compatibility matrices for your device for information about the latest certification ID entries (CIDs) for the combination of the ZBOSS stack v3.3 (or newer), the |NCS| version, and Nordic Semiconductor's SoCs:
```
This will work better, since the reader will be inside the TechDocs documentation anyway, so we don't need to draw attention to documentation platform changing when clicking one of the links.